ui/dashboards/src/components/PanelDrawer/PanelTypeSelect.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Select, SelectProps, MenuItem } from '@mui/material';
+import { useListPluginMetadata } from '@perses-dev/plugin-system';
+
+export type PanelTypeSelectProps = Omit<SelectProps<string>, 'children'>;
+
+/**
+ * Displays a MUI Select input for selecing a Panel type. Queries the plugin system to get the list of all panel types.
+ */
+export function PanelTypeSelect(props: PanelTypeSelectProps) {
+  const { value: propValue, ...others } = props;
```

Comment:
I'm down to keep calling this `value` within this component, but is there something you're trying to distinguish from?